src/helpers/charge_point/connector.ts

```diff
@@ -111,7 +111,7 @@ export default class AppChargePointConnector {
     const payload = {
       idTag: stoppingTransaction?.idTag,
       meterStop: Math.floor(this.currentEnergyWatt),
-      connectorId: stoppingTransaction?.connectorId,
+      // connectorId: stoppingTransaction?.connectorId,
```

Comment:
Uncommit that please we should put connectorId on payload.

<img width="557" alt="image" src="https://github.com/MuhammedAlmaz/ma-ocpp-simulator/assets/17336091/196dc4dc-7641-4478-8df3-964d1b3522a1">

Reply:
I commented the one in StopTransaction not in meterValues